calculator.go

```diff
@@ -1,13 +1,89 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(nums ...float64) float64 {
+	var result float64 = nums[0]
+
+	for _, n := range nums[1:] {
+		result = result + n
+	}
+	return result
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(nums ...float64) float64 {
```

Comment:
Nice! But this would allow us to pass zero or one arguments to Subtract, which doesn't seem right, does it? Could we write the parameter list in such a way that there _must_ be at least two arguments, but possibly more?

Reply:
I _think_ I have done this in 942a3eb and I've done this for all the operations that did accept multiple params. 
That said, in Divide I now need to error check twice to catch the division by 0, for some reason, it does not look right to me. Is this something I should worry about?